paper/paper.md

```diff
@@ -39,22 +39,22 @@ header-includes: |
 where $f: \mathbb{R}^n \to \mathbb{R}$ and $c: \mathbb{R}^n \to \mathbb{R}^m$ are continuously differentiable, and $h: \mathbb{R}^n \to \mathbb{R} \cup \{+\infty\}$ is lower semi-continuous.
 The nonsmooth objective $h$ can be a *regularizer* such as a sparsity-inducing penalty, model simple constraints such as $x$ belonging to a simple convex set, or be a combination of both.
 All $f$, $h$ and $c$ can be nonconvex.
-Together with the companion library [ShiftedProximalOperators.jl](https://github.com/JuliaSmoothOptimizers/ShiftedProximalOperators.jl) described below, RegularizedOptimization.jl provides a modular and extensible framework for solving \eqref{eq:nlp}, and developing novel solvers.
+RegularizedOptimization.jl provides a modular and extensible framework for solving \eqref{eq:nlp}, and developing novel solvers.
 Currently, the following solvers are implemented:
 
 - **Trust-region solvers TR and TRDH** [@aravkin-baraldi-orban-2022;@leconte-orban-2023]
 - **Quadratic regularization solvers R2, R2DH and R2N** [@diouane-habiboullah-orban-2024;@aravkin-baraldi-orban-2022]
-- **Levenberg-Marquardt solvers LM and LMTR** [@aravkin-baraldi-orban-2024] used when $f$ is a least-squares residual
+- **Levenberg-Marquardt solvers LM and LMTR** [@aravkin-baraldi-orban-2024].
 - **Augmented Lagrangian solver AL** [@demarchi-jia-kanzow-mehlitz-2023].
 
 All solvers rely on first derivatives of $f$ and $c$, and optionally on their second derivatives in the form of Hessian-vector products.
-If second derivatives are not available or too costly to compute, quasi-Newton approximations can be used.
-In addition, the proximal mapping of the nonsmooth part $h$, or adequate models thereof, must be evaluated.
+If second derivatives are not available, quasi-Newton approximations can be used.
+The proximal mapping of the nonsmooth part $h$, or adequate models thereof, must be evaluated.
```

Comment:
Si on a place pour 2 mots, Je remettrais "In addition" ici car sinon, la phrase tombe un peu à plat.